redhat-operators/amq-online/amq-online.1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1030 @@
+    mediatype: image/svg+xml
```

Comment:
I believe the base64 for the icon is actually a PNG rather than svg

Reply:
Indeed. I will fix this.

Reply:
I pushed a change, switching to PNG. I will also try to get an SVG version for that same image.

Reply:
This icon is the Red Hat logo (the hat). Will this be the logo for AMQ Online?